### PR TITLE
Fix drone docker wrapper #34

### LINKED ISSUE #34: Fix bad env var in drone docker wrapper
It appears the env variable declarations in the shell wrapper are off in the `CREATE_REPOSITORY` section on L7.

### PR DESCRIPTION
**Pull request by mbeacom**
- [x] Resolves #34 - Bad environment variable being set on L7.

### CODE REVIEW

**Review by tboerger — APPROVED** [review]

**Comment by tomreay** [comment]
Hi, is there a plan to release this?  The plugin seems to be broken because of this

**Comment by tboerger** [comment]
> Hi, is there a plan to release this? The plugin seems to be broken because of this

Generally I was waiting for another review by one of the other maintainers. I have merged this now to get it rolling.

**Comment by tboerger** [comment]
Damn, just realized that this had been a bad idea. The ECR plugin have been merged into https://github.com/drone-plugins/drone-docker.

**Pull request merged**
